src/arrays.js

```diff
@@ -1,61 +1,74 @@
 const getNthElement = (index, array) => {
-  // your code here
+  const newIndex = index < array.length ? index : index - array.length;
+  return array[newIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join();
 };
 
 const csvStringToArray = string => {
-  // your code here
+  return string.split();
```

Comment:
For the split method to work you need to pass it a character to split the string around, in this case a comma:
```
  return string.split(',');
```